docs/integrations/destinations/snowflake.md

```diff
@@ -270,7 +270,8 @@ Otherwise, make sure to grant the role the required permissions in the desired n
 ## Changelog
 
 | Version         | Date       | Pull Request                                               | Subject                                                                                                                                             |
-| :-------------- | :--------- | :--------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
+|:----------------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.1.1           | 2023-06-21 | [\#27342](https://github.com/airbytehq/airbyte/pull/27342) | Passes in the serialized record data instead of full record metadata with the CSV writer                                                            |
```

Comment:
Maybe we should be using v1.2.0, and say something about "faster because parallelism"

Reply:
snowflake goes brrrrrrr